docker/configs/php/xdebug.ini

```diff
@@ -5,6 +5,9 @@ xdebug.trace_output_name=trace.%R.%u
 xdebug.profiler_output_name=profile.%R.%u
 xdebug.output_dir=/shared
 
+xdebug.log = /shared/xdebug.log
```

Comment:
What do you think about keeping this outside of the shared directory and just putting it to /tmp and adding it to be tailed to the docker logs: https://github.com/juliushaertl/nextcloud-docker-dev/blob/master/docker/bin/bootstrap.sh#L343

That way we stick more to the usual log handling for docker and also do not conflict when multiple containers are running.

Reply:
That is a good idea.

I would suggest using a FIFO then to prevent storage overflow inside the container if you do not mind. I would then adopt that accordingly. Just give me a short ping if you see any issues.

Reply:
Sure, sounds good 👍

Reply:
I tried to realize that FIFO thing. The problem is that the FIFO seems to be useable only by one user. When trying to access (also only reading) the FIFO as a user who does not own the FIFO, there is a permission issue rosen.

I was not able to make it work between multiple users. Thus, I stuck with the classical file approach. Let's keep it that way. It will make things work at least.